content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,60 @@
+
+### CEL for Admission Control ([KEP-3488](https://kep.k8s.io/3488))
+
+Integrating Common Expression Language (CEL) for admission control in Kubernetes introduces a more dynamic and expressive way of evaluating admission requests. This feature allows complex, fine-grained policies to be defined and enforced directly through the Kubernetes API, enhancing security and governance capabilities without compromising performance or flexibility.
+
+CEL's addition to Kubernetes admission control empowers cluster administrators to craft intricate rules that can evaluate the content of API requests against the desired state and policies of the cluster without resorting to Webhook-based access controllers. This level of control is crucial for maintaining the integrity, security, and efficiency of cluster operations, making Kubernetes environments more robust and adaptable to various use cases and requirements.
+
+### Container Resource based Pod Autoscaling ([KEP-1610](https://kep.k8s.io/1610))
+
+HPA ContainerResource type metric graduates to stable in v1.30. It allows us to configure Autoscaling based on resource usage of individual containers. 
+See [our previous blog](/blog/2023/05/02/hpa-container-resource-metric/) for further details.
+
+## Want to know more?
+
+Deprecations are announced in the Kubernetes release notes. You can see the announcements of pending deprecations in the release notes for:
+
+* [Kubernetes v1.26](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md#deprecation)
+* [Kubernetes v1.27](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#deprecation)
+* [Kubernetes v1.28](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md#deprecation)
+* [Kubernetes v1.29](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.29.md#deprecation)
+
+
+We will formally announce the deprecations that come with [Kubernetes v1.30](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.30.md#deprecation) as part of the CHANGELOG for that release.
+
+For information on the deprecation and removal process, refer to the official Kubernetes [deprecation policy](/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api) document.
```

Comment:
> Not sure if this fits the theme, what do folks think?

It should likely be removed since we're no longer discussing deprecations--I will check with the team.